src/ft2font_wrapper.cpp

```diff
@@ -552,6 +555,76 @@ static PyObject *PyFT2Font_get_kerning(PyFT2Font *self, PyObject *args)
     return PyLong_FromLong(result);
 }
 
+const char *PyFT2Font_get_fontmap__doc__ =
+    "_get_fontmap(self, string)\n"
+    "--\n\n"
+    "Get a mapping between characters and the font that includes them.\n"
+    "A dictionary mapping unicode characters to PyFT2Font objects.";
+static PyObject *PyFT2Font_get_fontmap(PyFT2Font *self, PyObject *args, PyObject *kwds)
+{
+    PyObject *textobj;
+    const char *names[] = { "string", NULL };
+
+    if (!PyArg_ParseTupleAndKeywords(
+            args, kwds, "O:_get_fontmap", (char **)names, &textobj)) {
+        return NULL;
+    }
+
+    std::set<FT_ULong> codepoints;
+    size_t size;
+
+    if (PyUnicode_Check(textobj)) {
+        size = PyUnicode_GET_LENGTH(textobj);
+#if defined(PYPY_VERSION) && (PYPY_VERSION_NUM < 0x07040000)
+        // PyUnicode_ReadChar is available from PyPy 7.3.2, but wheels do not
+        // specify the micro-release version, so put the version bound at 7.4
+        // to prevent generating wheels unusable on PyPy 7.3.{0,1}.
+        Py_UNICODE *unistr = PyUnicode_AsUnicode(textobj);
+        for (size_t i = 0; i < size; ++i) {
+            codepoints.insert(unistr[i]);
+        }
+#else
+        for (size_t i = 0; i < size; ++i) {
+            codepoints.insert(PyUnicode_ReadChar(textobj, i));
+        }
+#endif
+    } else {
+        PyErr_SetString(PyExc_TypeError, "String must be str");
```

Comment:
```suggestion
        PyErr_SetString(PyExc_TypeError, "string must be str");
```
No big deal, but in case you still push a new version...

Reply:
I took this as matching the variable name seems better than matching grammer here.